src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -500,33 +489,31 @@ public Command testTurretCommand(double degrees) {
   }
 
   private void aimTurret() {
-    double limelightTX = LimelightHelpers.getTX("limelight");
-    if (limelightTX != 0
-        && LimelightHelpers.getFiducialID("limelight")
-            == ((robotStateManager.getAllianceColor()
-                    == AllianceColor
-                        .BLUE) // Default to red because that's the color on our test field
-                ? Constants.TurretConstants.SPEAKER_TAG_ID_BLUE
-                : Constants.TurretConstants.SPEAKER_TAG_ID_RED)) {
-      // X & Rotation
-      setTurretPos(Math.toRadians(limelightTX) + turretPosition);
-
-      // Y & Pitch control
-      double limelightTY = LimelightHelpers.getTY("limelight");
-      double distanceToTag = tyToDistanceFromTag(limelightTY);
-      tagDistanceEntry.log(distanceToTag);
-      setPitchPos(distanceToShootingPitch(distanceToTag));
-
-      if (Math.toRadians(limelightTX) + turretPosition
-              > Math.toRadians(Constants.TurretConstants.TURRET_MAX_ANGLE_DEGREES)
-          || Math.toRadians(limelightTX) + turretPosition
-              < Math.toRadians(Constants.TurretConstants.TURRET_MIN_ANGLE_DEGREES)) {
-        // TODO: Make turret rotate the drivebase if necessary and the driver thinks it's a good
-        // idea
+    if (visionSubsystem != null) {
+      int tagID =
+          ((robotStateManager.getAllianceColor()
```

Comment:
Side note: I imagine we will do this type of thing (ask "is the alliance blue" or "is the alliance red") to warrant having accessor methods to that effect, e.g. `robotStateManager.isAllianceBlue()` and `...isAllianceRed()`.